app/src/main/java/com/github/se/gomeet/ui/mainscreens/events/MyEventInfo.kt

```diff
@@ -266,6 +292,37 @@ fun MyEventInfo(
   if (showShareEventDialog) {
     ShareDialog("Event", eventId, onDismiss = { showShareEventDialog = false })
   }
+
+  if (showDeleteEventDialog) {
+    DeleteEventDialog(
+        onConfirm = {
+          coroutineScope.launch {
+            myEvent.value!!.participants.forEach { participant ->
+              userViewModel.gotKickedFromEvent(eventId, participant)
+
+              if (participant == currentUser.value!!.uid) {
+                userViewModel.userDeletesEvent(eventId, participant)
+              }
+            }
+
+            myEvent.value!!.pendingParticipants.forEach { pendingParticipant ->
+              userViewModel.invitationCanceled(eventId, pendingParticipant)
+            }
+
+            val allUsers = userViewModel.getAllUsers()
+
+            allUsers!!.forEach { user ->
+              if (user.myFavorites.contains(eventId)) {
```

Comment:
Shouldn't each users' `joinedEvents` list also get checked and potentially updated here upon the deletion of the events?

Reply:
My bad, I realised that `gotKickedFromEvent` handles that.